### PR TITLE
Navigation to map when tapped access

### LINKED ISSUE #680: Add `Access` implementation in about screen
## Kind (Required)

- Proposal / Discussion

 ## Overview (Required)

- I'm thinking about launching Google Map with an implicit intent.
-  https://github.com/DroidKaigi/conference-app-2020/blob/7f29bcb303e6056a933b9d687b21c9804640f647/feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt#L89

 ## Links	

-

### PR DESCRIPTION
**Pull request by youmitsu**
## Issue
- close #680 

## Overview (Required)
- On about page, navigation to map when tapped access button.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt [hunk 1]
```diff
@@ -87,7 +89,13 @@ class AboutFragment : Fragment(R.layout.fragment_about), Injectable {
                 aboutItemFactory.create(
                     getString(R.string.about_item_access)
                 ) {
-                    // TODO go access-page
+                    val intent = Intent(
+                        Intent.ACTION_VIEW,
+                        Uri.parse("geo:35.6219252,139.7190626?q=TOCビル")
+                    ).setPackage("com.google.android.apps.maps")
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt]: I want to use implicit intent. So I think this is not needed 👀 
If you have any opinion please tell me.

File: feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt [hunk 2]
```diff
@@ -87,7 +89,13 @@ class AboutFragment : Fragment(R.layout.fragment_about), Injectable {
                 aboutItemFactory.create(
                     getString(R.string.about_item_access)
                 ) {
-                    // TODO go access-page
+                    val intent = Intent(
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt]: Currently, we use `SystemViewModel` for launching intent. 🙏

Comment (youmitsu) [reply feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt]: Fixed.

File: feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt (hunk 1 above)
Comment (youmitsu) [reply feature/about/src/main/java/io/github/droidkaigi/confsched2020/about/ui/AboutFragment.kt]: Fixed.

**Comment by youmitsu** [comment]
@takahirom 
I fixed!
Could you check that again, please?

**Pull request merged**